src/main/java/duke/command/DoneCommand.java

```diff
@@ -0,0 +1,38 @@
+package duke.command;
+
+import duke.*;
```

Comment:
It's a good practice to avoid using * when importing. You should only import statements that you are going to use! 😄